lib/elliptic/curve/mont.js

```diff
@@ -147,6 +158,10 @@ Point.prototype.mulAdd = function mulAdd() {
   throw new Error('Not supported on Montgomery curve');
 };
 
+Point.prototype.eq = function eq(other) {
+  return this.getX().cmp(other.getX()) === 0;
```

Comment:
If it calms you down, the `normalized form puts z as 1 huh?` was one of those questions that already had an answer.